src/timeMachine/utils/rawFluxDataTable.ts

```diff
@@ -29,6 +29,18 @@ export const parseFromFluxResults = (
   const groupSet = new Set(fluxGroupKeyUnion)
   let max = 0
 
+  // checks whether the string is valid JSON object or not
+  const isJsonObject = jsonString => {
+    try {
+      const object = JSON.parse(jsonString)
+      if (object && typeof object === 'object') {
+        return object
```

Comment:
I think this should `return true`. An object is truthy, but it's good to be explicit in these cases and not rely on weird JS truthiness rules.

Reply:
Got it, fixed!